common/src/main/java/org/apache/comet/parquet/Native.java

```diff
@@ -257,7 +257,8 @@ public static native long initRecordBatchReader(
       byte[] requiredSchema,
       byte[] dataSchema,
       String sessionTimezone,
-      int batchSize);
+      int batchSize,
+      boolean pushdownFilters);
```

Comment:
This is already implied if the filter parameter is null.

Reply:
Thanks. I will update this today.